minerva-core/src/main/java/org/geneontology/minerva/BlazegraphMolecularModelManager.java

```diff
@@ -570,7 +573,8 @@ protected void loadModel(IRI modelId, boolean isOverride) throws OWLOntologyCrea
 				graphs.close();
 				RepositoryResult<Statement> statements =
 						connection.getStatements(null, null, null, false, new URIImpl(modelId.toString()));
-				boolean minimal = true;
+				//boolean minimal = true;
+				boolean minimal = false;
```

Comment:
You had changed this to `true` on March 9 in 35912e54f6030b3085ffb793f53d2e154f9a756b. Was there a reason that needs to be addressed a different way?

Reply:
Honestly I can't remember why.  Looking at what it is doing, it makes sense for it to be set to false for the loadModel method and true for the loadModelABox method.  I think I didn't really understand the difference earlier.    

I will make that change in the next commit.  

It looks like the only thing that is using the loadModelABox method now is the createInferredModel method in CoreMolecularModelManager.  And that seems fine as it loads all the rules up from a separate reference ontology.  Its alarming that these two methods are intertwined yet the two problems I found appear to me to be separate issues..

Reply:
>  I think I didn't really understand the difference earlier.

That's totally understandable, since it isn't really documented. There is a little bit of explanation in the superclass, but this is a dangerous method and we should make that more clear.